packages/frontend/src/index.tsx

```diff
@@ -18,7 +18,7 @@ root.render(
       <UserProvider>
         <ApiProvider>
           <LocalizationProvider dateAdapter={AdapterLuxon}>
-            <SnackbarProvider maxSnack={3}>
+            <SnackbarProvider maxSnack={3} autoHideDuration={10000}>
```

Comment:
Unless they've asked for this I'd revert this, autohiding notifications is a bit of an accessibility rabbit hole.  You've gotta ensure all messages (of all supported lengths) have sufficient time on screen to be read by people of all abilities.